adafruit_ht16k33/segments.py

```diff
@@ -256,3 +255,50 @@ def _put(self, char, index=0):
         else:
             return
         self._set_buffer(index, NUMBERS[character])
+
+class BigSeg7x4(Seg7x4):
+    """Numeric 7-segment display. It has the same methods as the alphanumeric display, but only
+       supports displaying a limited set of characters."""
+    def __init__(self, i2c, address=0x70, auto_write=True):
+        super().__init__(i2c, address, auto_write)
+        self.colon = Colon(self, 2)
+
+    @property
+    def ampm(self):
+        """The AM/PM indicator."""
+        return bool(self._get_buffer(0x04) & 0x10)
+
+    @ampm.setter
+    def ampm(self, value):
+        current = self._get_buffer(0x04)
+        if value:
+            self._set_buffer(0x04, current | 0x10)
+        else:
+            self._set_buffer(0x04, current & ~0x10)
+        if self._auto_write:
+            self.show()
+
+#pylint: disable=W0212
```

Comment:
We prefer you use the spelled out version of the disables. Please update this to reflect the error in that format. For example: `# pylint: disable=too-many-arguments`.